### PR TITLE
Introduce RHEL7 C2S profile (derived from CIS requirements)

### PR DESCRIPTION
**Pull request by shawndwells**
- Introduce C2S profile

- Link into Make

- Update tables to generate policy mappings using two different formats for now. Let users decide what they like best:
http://people.redhat.com/swells/scap-security-guide/RHEL/7/output/table-rhel7-cisrefs-c2s.html
http://people.redhat.com/swells/scap-security-guide/RHEL/7/output/table-rhel7-cisrefs.html

- Add CIS policy mappings, originating from 
https://benchmarks.cisecurity.org/tools2/linux/CIS_Red_Hat_Enterprise_Linux_7_Benchmark_v1.1.0.pdf

- Update OVAL package templates for multi_platform_rhel

### CODE REVIEW

**Comment by shawndwells** [comment]
Noticed the merge conflicts, rebased and merged.

Bump for review.

File: RHEL/7/input/oval/package_aide_installed.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<def-group>
+ <!-- THIS FILE IS GENERATED by create_package_installed.py.  DO NOT EDIT.  -->
+  <definition class="compliance" id="package_aide_installed"
+  version="1">
+    <metadata>
+      <title>Package aide Installed</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
```
Comment (iankko) [inline RHEL/7/input/oval/package_aide_installed.xml]: This is just RHEL-7 platform specific.

File: RHEL/7/input/oval/package_autofs_removed.xml [hunk 2]
```diff
@@ -5,10 +5,10 @@
     <metadata>
       <title>Package autofs Removed</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_rhel</platform>
```
Comment (iankko) [inline RHEL/7/input/oval/package_autofs_removed.xml]: And another OVAL placed into the very same directory is instructed to be multi-platform. Can you explain why?

File: RHEL/7/input/oval/package_telnet_removed.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<def-group>
+ <!-- THIS FILE IS GENERATED by create_package_removed.py.  DO NOT EDIT.  -->
+  <definition class="compliance" id="package_telnet_removed"
+  version="1">
+    <metadata>
+      <title>Package telnet Removed</title>
+      <affected family="unix">
+        <platform>multi_platform_rhel</platform>
```
Comment (iankko) [inline RHEL/7/input/oval/package_telnet_removed.xml]: Again. Compare this one. With the following ``` RHEL/7/input/oval/package_vsftpd_installed.xml ``` one. Why such inconsistency?

File: RHEL/7/input/xccdf/services/obsolete.xml [hunk 4]
```diff
@@ -375,7 +378,7 @@ accidental (or intentional) activation of tftp services.
 </rationale>
 <ident cce="RHEL7-CCE-TBD" />
 <oval id="package_tftp-server_removed" />
-<ref nist="AC-17(8),CM-7" disa="305"/>
+<ref nist="AC-17(8),CM-7" disa="305" cis="2.1.7" />
```
Comment (iankko) [inline RHEL/7/input/xccdf/services/obsolete.xml]: This should be cis="2.1.8".

File: RHEL/7/input/xccdf/services/obsolete.xml [hunk 5]
```diff
@@ -392,6 +395,7 @@ for TFTP (such as a boot server). In that case, use extreme caution when configu
 the services.</rationale>
 <ident cce="" />
 <oval id="package_tftp_removed" />
+<ref cis="2.1.8" />
```
Comment (iankko) [inline RHEL/7/input/xccdf/services/obsolete.xml]: This should be cis="2.1.7"

**Comment by iankko** [comment]
I am going to merge this (the benefit of having this included overcomes the importance of the issues found during the review -- we can fix them on the way).

Overall sanity check:
* verified ```make validate``` passes for both ```RHEL/6``` and ```Fedora``` products on both (pre-OVAL-5.11 and post-OVAL-5.11) versions of openscap.

ACK from me.

**Pull request merged**